buildSrc/build.gradle

```diff
@@ -1,30 +1,5 @@
-apply plugin: 'idea'
-apply plugin: 'groovy'
-
-repositories { jcenter() }
-
 dependencies {
-    compile gradleApi()
-    //TODO SF use jcabi to edit issues after the release so that they have the milestone attached
-    //compile "com.jcabi:jcabi-github:0.17"
-    compile "com.googlecode.json-simple:json-simple:1.1.1@jar"
+    //needed by JavadocExclude
+    //TODO find out if we really need JavadocExclude. Why don't we just exclude internal packages using Gradle API (Javadoc.exclude)?
     compile files("${System.properties['java.home']}/../lib/tools.jar")
```

Comment:
I don't think that's possible at this time, or that can be brittle, the reason I believe is that gradle invoke javadoc file by file. This mode disable some features of `javadoc` tool, and may even make the `javadoc` tool fail if there's a reference to some excluded file within the javadoc itself.